openstax/settings/base.py

```diff
@@ -216,3 +214,32 @@
 
 # WAGTAIL SETTINGS
 WAGTAIL_SITE_NAME = 'openstax'
+
+##################################
+#        ACCOUNTS SETTINGS       #
```

Comment:
I would remove these settings from the base settings file (it's what was causing my error with accounts) - the keys/urls should be defined in either dev.py or local.py. I was changing these keys, but they were being overridden by dev.py.